provider/provider.go

```diff
@@ -77,6 +77,8 @@ func DatabricksProvider() *schema.Provider {
 			"databricks_global_init_script": workspace.ResourceGlobalInitScript(),
 			"databricks_notebook":           workspace.ResourceNotebook(),
 			"databricks_workspace_conf":     workspace.ResourceWorkspaceConf(),
+
+			"databricks_live_table": compute.ResourceLiveTable(),
```

Comment:
what do we have in databricks cli? `databricks pipeline deploy` or `databricks live-tables deploy`? preferably we should have consistent naming with databricks cli, when it's possible. technically, `databricks_live_table` would define 1+ table in the artifact, so i wonder about the naming.

Reply:
Currently, it is `pipelines`, but the plan is to rename it. I suppose we can start with this. We can always register another provider name with the same resource definition.

Reply:
Final answer: the resource will be called `pipeline`, and it will manage one or many live tables.